packages/server/__snapshots__/scaffold_spec.js

```diff
@@ -587,7 +587,7 @@ exports['lib/scaffold .plugins creates pluginsFile when pluginsFolder does not e
 /**
  * @type {Cypress.PluginConfig}
  */
-// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
@karlhorky this is a snapshot used in our internal tests, not the actual scaffolding code. The scaffolding is in another repository: https://github.com/cypress-io/cypress-example-kitchensink/blob/319d0d3913ec7e002e000381a8bdee89969ed543/cypress/plugins/index.js

However, I'm still not sure if this makes sense, since we are scaffolding a `.js` file...

I am going to close this PR in anticipation of you submitting a PR to `cypress-example-kitchensink`.

Reply:
Ok, opened over there: https://github.com/cypress-io/cypress-example-kitchensink/pull/502

Will continue discussion over there, but just as a short note, this `@typescript-eslint/no-unused-vars` rule can be enabled for JS files as well, which is why I have this error.